package.json

```diff
@@ -97,6 +97,7 @@
     "dotenv": "^16.4.7",
     "glob": "^10.4.5",
     "sha.js": "^2.4.11",
+    "sql-formatter": "^15.5.2",
```

Comment:
I see now that we depend on `@sqltools/fomatter` above, can we use that instead of adding `sql-formatter` as a new dependency? It's a significantly smaller package ([75kB](https://www.npmjs.com/package/@sqltools/formatter) vs [2.61MB](https://www.npmjs.com/package/sql-formatter)).

Another option to consider is installing this as an [optional peer dependency](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#peerdependenciesmeta).

I'm concerned about the increase in size from requiring this new package. It's going to make TypeORM larger to install (20.5MB to 23MB), and most installs won't need a formatting option.

Reply:
I need to test it, but I think something like:
ac1f181b53cc4dae4cc11a22ef5beb3caca80398
It uses the same package now as we use to generate the migrations.